FirebaseAuth/Sources/Swift/MultiFactor/MultiFactor.swift

```diff
@@ -114,6 +114,7 @@ import Foundation
             }
           }
         }
+        return
```

Comment:
Without the `return`, it falls through to line 121 where the fatal error on line 123 will occur. The issue is the fall-through.

Reply:
Line 121 onwards should only run for phone MFA enrollment, not TOTP MFA enrollment.

Reply:
Good catch!